src/harness/testcases/WINNF_FT_S_MCP_testcase.py

```diff
@@ -0,0 +1,328 @@
+#    Copyright 2017 SAS Project Authors. All Rights Reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+
+import json
+import os
+import sas
+import sas_objects
+import constraint_testcase
+from util import winnforum_testcase, configurable_testcase, writeConfig, \
+  loadConfig, generateCpiRsaKeys, generateCpiEcKeys, convertRequestToRequestWithCpiSignature
+
+class MultiConstraintProtectionTestcase(constraint_testcase.ConstraintTestcase):
+
+  def setUp(self):
+    self._sas, self._sas_admin = sas.GetTestingSas()
+    self._sas_admin.Reset()
+
+  def tearDown(self):
+    pass
+
+  def generate_MCP_1_default_config(self,filename):
+    """ Generates the WinnForum configuration for MCP.1. """
+    # Load Grant requests
+    grant_record_1 = json.load(
+        open(os.path.join('testcases', 'testdata', 'grant_0.json')))
+    grant_record_2 = json.load(
+        open(os.path.join('testcases', 'testdata', 'grant_1.json')))
+    grant_record_3 = json.load(
+        open(os.path.join('testcases', 'testdata', 'grant_2.json')))
+    grant_record_4 = json.load(
+        open(os.path.join('testcases', 'testdata', 'grant_3.json')))
+    
+    #Load CBSD
+    device_1 = json.load(
+        open(os.path.join('testcases', 'testdata', 'device_a.json')))
+    device_2 = json.load(
+        open(os.path.join('testcases', 'testdata', 'device_b.json')))
+    device_3 = json.load(
+        open(os.path.join('testcases', 'testdata', 'device_c.json')))
+    device_4 = json.load(
+        open(os.path.join('testcases', 'testdata', 'device_d.json')))
+
+    # Load GWPZ Record
+    gwpz_record_1 = json.load(
+        open(os.path.join('testcases', 'testdata', 'gwpz_record_0.json')))
+
+    # Load FSS record
+    fss_record_1 =  json.load(
+        open(os.path.join('testcases', 'testdata', 'fss_record_0.json')))
+    fss_record_2 =  json.load(
+        open(os.path.join('testcases', 'testdata', 'fss_record_1.json')))
+
+    # Load ESC record
+    esc_record_1 =  json.load(
+        open(os.path.join('testcases', 'testdata', 'esc_sensor_record_0.json')))
+
+
+    # Create the actual config.
+    config = {
+        'cbsdRecords': [{'registrationRequests':[device_1,device_2],'grantRequests':[grant_record_1,grant_record_2]},
```

Comment:
Please use a format consistent with existing config files. Please also fix the formatting so that this is more readable.